tests/snapshots/tests.contrib.starlette.test_starlette.test_schematization[service_schema2]_36.json

```diff
@@ -50,10 +50,15 @@
          "http.url": "http://testserver/sub-app/hello/name",
          "http.useragent": "testclient",
          "http.version": "1.1",
+         "runtime-id": "1b7f12e59e5044409a4d983ea7296950",
          "span.kind": "server"
        },
-       "duration": 288000,
-       "start": 1691164875685839000
+       "metrics": {
+         "_dd.top_level": 1,
```

Comment:
When module unloading is enabled this span is marked as top level. This is a behavior we need to investigate further. This could be fixing a subtle bug with peer services.